components/hab/src/command/origin/key/list.rs

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2016-2017 Chef Software Inc. and/or applicable contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use glob::glob;
+use std::path::Path;
+
+use error::Result;
+
+pub fn start(cache: &Path) -> Result<()> {
+    if let Some(pattern) = cache.join("*.pub").to_str() {
```

Comment:
Sounds fine. I don't think that these filenames have characters outside ASCII set.